packages/webdriverio/webdriverio.d.ts

```diff
@@ -25,7 +31,7 @@ declare namespace WebdriverIO {
          * The executed script is assumed to be synchronous and the result of evaluating the script is returned to
          * the client.
          */
-        execute: <T>(script: string | ((...arguments: any[]) => T), ...arguments: any[]) => Promise<T>;
+        execute: <T>(script: string | ExecuteFunction, ...arguments: any[]) => Promise<T>;
```

Comment:
ExecuteFunction should accept type otherwise browser.execute return value won't be set properly